### ISSUE TITLE
macOS build?

### ISSUE THREAD

**Issue by hoechenberger** [conda-forge/glfw-feedstock#3]
Hello, thank you for creating this package! Any chances to see a macOS build anytime soon?

**Comment by hmaarrfk** [comment]
I have a feeling mac was particularly troublesome with these graphics packages. We would appreciate help getting it up and running.

What is the usecase you have where you need this package running on mac if we may ask?

**Comment by hoechenberger** [comment]
@hmaarrfk [PsychoPy](https://www.psychopy.org) recently added support for GLFW as a modern alternative to pyglet and pygame. I would like to add `glfw` support to the PsychoPy [`conda-forge` package](https://github.com/conda-forge/psychopy-feedstock).

**Comment by hmaarrfk** [comment]
well we aren't opposed to it. we should definitely try once we get the latest update up and running. I can probably review and help you get the update and the mac osx through.

**Comment by hoechenberger** [comment]
Not sure if I have enough knowledge of this stuff, but we can give it a shot!

**Comment by hoechenberger** [comment]
Hm, this doesn't seem to be _too_ complicated, no? :)
https://www.glfw.org/docs/latest/compile.html#compile_deps_xcode

**Comment by hmaarrfk** [comment]
Want to make a PR for it, I would be able to commit to your branch

**Comment by hoechenberger** [comment]
@hmaarrfk I will be looking into this within the next few days, thanks!

**Comment by hoechenberger** [comment]
I filed a PR, #4 

It does compile correctly on macOS, however I'm not sure if it's actually being _packaged_ properly, as I get `conda inspect linkages` returns:
```
+ conda inspect linkages -p /usr/local/miniconda/conda-bld/glfw_1562336035562/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place glfw
WARNING: pyldd disagrees with ldd/otool. This will not cause any
WARNING: problems for this build, but please file a bug at:
WARNING: https://github.com/conda/conda-build
WARNING: and (if possible) attach file $PREFIX/lib/libglfw.3.3.dylib
WARNING: 
ldd/otool gives:
('libglfw.3.dylib', 'lib/libglfw.3.dylib')
('IOKit', '/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit')
('CoreFoundation', '/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation')
('CoreVideo', '/System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo')
('libSystem.B.dylib', '/usr/lib/libSystem.B.dylib')
('AppKit', '/System/Library/Frameworks/AppKit.framework/Versions/C/AppKit')
('CoreGraphics', '/System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics')
('CoreServices', '/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices')
('Foundation', '/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation')
('libobjc.A.dylib', '/usr/lib/libobjc.A.dylib')
pyldd gives:
('CoreFoundation', '/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation')
('libobjc.A.dylib', '/usr/lib/libobjc.A.dylib')
('CoreVideo', '/System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo')
('CoreGraphics', '/System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics')
('libSystem.B.dylib', '/usr/lib/libSystem.B.dylib')
('CoreServices', '/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices')
('Foundation', '/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation')
('IOKit', '/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit')
('AppKit', '/System/Library/Frameworks/AppKit.framework/Versions/C/AppKit')

Diffs
{('libglfw.3.dylib', 'lib/libglfw.3.dylib')}
Diffs
set()
glfw
----

system:
    AppKit (/System/Library/Frameworks/AppKit.framework/Versions/C/AppKit)
    CoreFoundation (/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation)
    CoreGraphics (/System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics)
    CoreServices (/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices)
    CoreVideo (/System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo)
    Foundation (/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation)
    IOKit (/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit)
    libSystem.B.dylib (/usr/lib/libSystem.B.dylib)
    libobjc.A.dylib (/usr/lib/libobjc.A.dylib)

not found:
    libglfw.3.dylib (not found)
```
And, funnily, Windows doesn't seem to build, either!

**Comment by hoechenberger** [comment]
Closed via #4.
